### PR TITLE
#160403302 Add more eslint rules

### PR DESCRIPTION
**Pull request by ronaldndirangu**
#### What does this PR do?
Alters ESLint configuration

#### Description of tasks to be completed
Ensure that all developers are in sync through standard text editor eslint configurations.

#### How should this be manually tested?
```sh
$ git clone https://github.com/andela/ah-titans-frontend.git
```
```sh
$ npm install
```
```sh
$ npm run lint
```
#### Any background context you want to provide?
No

#### What are the relevant Pivotal Tracker Stories?
[#160403302](https://www.pivotaltracker.com/story/show/160403302)

### CODE REVIEW

**Review by johnmusiu — APPROVED** [review]

**Review by migot01 — APPROVED** [review]

**Review by janetnim — APPROVED** [review]

File: .eslintrc.js [hunk 1]
```diff
@@ -2,11 +2,23 @@ module.exports = {
...
+    'react/prefer-stateless-function': 'off',
     'linebreak-style': ['error', 'unix'],
+    "prefer-destructuring": ["error", {"object": true, "array": false}],
+
     quotes: ['error', 'single'],
     semi: ['error', 'always'],
+    indent: ['error', 2],
 
     // override default options for rules from base configurations
     'comma-dangle': ['error', 'always'],
```
Comment (jmnyarega) [inline .eslintrc.js]: Remove this `comma-dangle` rule

File: .eslintrc.js [hunk 2]
```diff
@@ -2,11 +2,23 @@ module.exports = {
...
+    'arrow-body-style': 'off',
+    'react/require-default-props': 'off',
+    'react/jsx-boolean-value': 'off',
+    'react/prefer-stateless-function': 'off',
     'linebreak-style': ['error', 'unix'],
+    "prefer-destructuring": ["error", {"object": true, "array": false}],
+
     quotes: ['error', 'single'],
     semi: ['error', 'always'],
+    indent: ['error', 2],
```
Comment (jmnyarega) [inline .eslintrc.js]: Is this indentation `spaces` or `tabs` ?

Comment (ronaldndirangu) [reply .eslintrc.js]: It is spaces

**Review by dcDalin — APPROVED** [review]

File: .eslintrc.js (hunk 2 above)
Comment (jmnyarega) [reply .eslintrc.js]: Kindly let's use tabs

Comment (ronaldndirangu) [reply .eslintrc.js]: Does that mean I should enforce four spaces for the tabs?

Comment (ronaldndirangu) [reply .eslintrc.js]: I have made the changes.

**Pull request merged**